hint-report/file--home-hiro-Real-Microverse-BooksJavascript.html

```diff
@@ -1350,7 +1350,7 @@
                     </div>
                     
                         <div class="scan-overview--time-stamp">
-                            <p class="scan-overview__subheader">Date: <span id="scan-date" class="overview--time-stamp__time-stamp" data-date="2021-07-05T17:07:24.361Z">2021-07-05T17:07:24.361Z</span></p>
+                            <p class="scan-overview__subheader">Date: <span id="scan-date" class="overview--time-stamp__time-stamp" data-date="2021-07-08T10:29:03.189Z">2021-07-08T10:29:03.189Z</span></p>
```

Comment:
- [ ] According to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements), the current date should be displayed below the navigation bar, for this requirement, you should use the [Luxon](https://moment.github.io/luxon/docs/manual/install.html) library. See [Get the current date and time](https://moment.github.io/luxon/docs/manual/tour.html#get-the-current-date-and-time).

**Wireframe** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/124908431-2a27a480-df9e-11eb-85f0-fb779bee1763.png) | ![image](https://user-images.githubusercontent.com/49621279/124908459-36136680-df9e-11eb-8f96-6284df9ed0d1.png)